docs/content/tutorials/tutorial-streams.md

```diff
@@ -0,0 +1,134 @@
+configuration. The bundled configuration is in `conf-quickstart/tranquility/server.json`, which 
+you can modify for your own needs.
+
+The most important questions are:
+
+  * What should the dataset be called? This is the "dataSource" field of the "dataSchema".
+  * Which field should be treated as a timestamp? This belongs in the "column" of the "timestampSpec".
+  * Which fields should be treated as dimensions? This belongs in the "dimensions" of the "dimensionsSpec".
+  * Which fields should be treated as measures? This belongs in the "metricsSpec".
+
+Let's use a small JSON pageviews dataset as an example, with records like:
+
+```json
+{"time": "2000-01-01T00:00:00Z", "url": "/foo/bar", "user": "alice", "latencyMs": 32}
+```
+
+So the answers to the questions above are:
+
+  * Let's call the dataset "pageviews".
+  * The timestamp is the "time" field.
+  * Good choices for dimensions are the string fields "url" and "user".
+  * Good choices for measures are a count of pageviews, and the sum of "latencyMs". Collecting that 
+sum when we load the data will allow us to compute an average at query time as well.
+
+Now, edit the existing `conf-quickstart/tranquility/server.json` file by altering these 
+sections:
+
+  1. Change the key `"metrics"` under `"dataSources"` to `"pageviews"`
+  2. Alter these sections under the new `"pageviews"` key:
+  ```json
+  "dataSource": "pageviews"
+  ```
+
+  ```json
+  "timestampSpec": {
+       "format": "auto",
+       "column": "time"
+  }
+  ```
+
+  ```json
+  "dimensionsSpec": {
+       "dimensions": ["url", "user"]
+  }
+  ```
+
+  ```json
+  "metricsSpec": [
+       {"name": "views", "type": "count"},
+       {"name": "latencyMs", "type": "doubleSum", "fieldName": "latencyMs"}
+  ]
+  ```
+
+## Restarting the server
+
+Restart the server to pick up the new configuration file by CTRL-C out of Tranquility and starting it up again.
+
+## Sending data
+
+Let's send some data! We'll start with these three records:
+
+```json
+{"time": "2000-01-01T00:00:00Z", "url": "/foo/bar", "user": "alice", "latencyMs": 32}
+{"time": "2000-01-01T00:00:00Z", "url": "/", "user": "bob", "latencyMs": 11}
+{"time": "2000-01-01T00:00:00Z", "url": "/foo/bar", "user": "bob", "latencyMs": 45}
+```
+
+Druid streaming ingestion requires relatively current messages (relative to a slack time controlled by the 
+[windowPeriod](ingestion-streams.html#segmentgranularity-and-windowperiod) value), so you should 
+replace `2000-01-01T00:00:00Z` in these messages with the current time in ISO8601 format. You can 
+get this by running:
+
+```bash
+python -c 'import datetime; print(datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"))'
+```
+
+Update the timestamps in the JSON above, and save it to a file named `pageviews.json`. Then send 
+it to Druid by running:
+
+```bash
+curl -XPOST -H'Content-Type: application/json' --data-binary @pageviews.json http://localhost:8200/v1/post/pageviews
+```
+
+Which will print something like:
```

Comment:
nit: lower case w in which - or, start a new sentence: "This will print something like ..."